src/js/pie.js

```diff
@@ -21,19 +21,25 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
     // 4. update the pie when the data set changes, including adding or removing slices
 
     fluid.defaults("floe.chartAuthoring.pieChart.pie", {
-        gradeNames: ["floe.chartAuthoring.totalRelaying", "floe.d3ViewComponent",  "autoInit"],
+        gradeNames: ["floe.chartAuthoring.valueBinding","floe.chartAuthoring.totalRelaying", "floe.d3ViewComponent"],
```

Comment:
Cheers, that gist is an awesome guide! Sorry for the waste of time. The formatting of your linting messages is still puzzling though, mine appear as follows:

   Gruntfile.js
     15 |module.exports = function(grunt) {
                                  ^ Missing space after 'function'.